rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -3231,4 +3252,91 @@ private boolean isXmlWorkflow(InputPart fileInputPart) {
                 .contains(MediaType.APPLICATION_XML.toLowerCase());
     }
 
+    @Override
+    public RestPage<String> getTaskIds(String sessionId, long from, long to, boolean mytasks, boolean running,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 9 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)